frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -27,7 +27,6 @@ local MODE = {
     book_time_to_read = 6,
     chapter_time_to_read = 7,
     frontlight = 8,
-    mem_usage = 9,
```

Comment:
That's what I proposed in the initial discussion and why I hold off merging this footer info into koreader for a long time. I eventually ended up merging it because no one had the time to implement the auto restart reminder feature. If you want to implement this, that would be great.